src/policy/policy.cpp

```diff
@@ -289,6 +332,12 @@ bool IsWitnessStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
             }
         }
     }
+
+    // If any inputs commit to an annex, all inputs must commit to an annex.
+    if (annex_input_count > 0 && tx.vin.size() != tx.vin.size()) {
```

Comment:
should this be `if (annex_input_count > 0 && tx.vin.size() != annex_input_count)`?

Reply:
fixed